experiments/run_finetuning.py

```diff
@@ -62,6 +62,21 @@ def get_train_dataloader(self) -> DataLoader:
 
     def get_eval_dataloader(self, _) -> DataLoader:
         return self.test_loader
+    
+    def compute_loss(self, model, inputs, return_outputs=False):
```

Comment:
As this is most of the calculation for ppl can we make this one and our `gpu_utils` implementation consistent, and add a test? Better still would be to call a common subroutine from here and from the the ppl calc implementation. NB Pashmin'as ppl refactoring PR should go in first.

Reply:
Yes, the perplexity calculation should have the same changes, good idea to combine them